coding/cpp_style_guide.rst

```diff
@@ -1984,432 +1982,64 @@ We should, of course, strive to minimize the amount of such information.
 
 .. _style-guide-cpp-6-1:
 
-6-1. Multiple statements per line SHOULD NOT be used.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: cpp
+6-1. We follow the clang-format Google style for code layout with a few exceptions.
```

Comment:
If this goes forward, we need a primer on what the new style is, and how it differs from the current style.